src/tests/ftest/pool/management_race.py

```diff
@@ -110,7 +111,8 @@ def test_pool_management_race(self):
                 del_recreate=False, test_pool=t_pool, thread_num=index, test_loop=test_loop)
         # Setup the thread manager for del_and_recreate_pool
         thread_manager.add(
-            del_recreate=True, test_pool=self.pool[pool_number], thread_num=index, test_loop=test_loop)
+            del_recreate=True, test_pool=self.pool[pool_number],
+            thread_num=index, test_loop=test_loop)
```

Comment:
The pylint complaint here has some merit.  What is the expected value of index after the loop completes?  What if self.pool is empty.  Better would be an explicit value.
```suggestion
            thread_num=len(self.pool), test_loop=test_loop)
```

Reply:
I agree this complaint has merit and I hope that next time someone is working on this test with context of what it's trying to do that they address this.